operator/rpc_client.go

```diff
@@ -216,3 +236,22 @@ func (c *AggregatorRpcClient) SendSignedOperatorSetUpdateToAggregator(signedOper
 		return nil
 	}, signedOperatorSetUpdateMessage)
 }
+
+func (c *AggregatorRpcClient) GetAggregatedCheckpointMessages(fromTimestamp, toTimestamp uint64, checkpointMessages *messages.CheckpointMessages) error {
```

Comment:
```suggestion
func (c *AggregatorRpcClient) GetAggregatedCheckpointMessages(fromTimestamp, toTimestamp uint64) (*messages.CheckpointMessages, error) {
```
Since its a getter lets just return `*messages.CheckpointMessages`.

There will be no need to do smth like this before calling then.
`var checkpointMessages messages.CheckpointMessages
`

this can be done inside the function itself

Reply:
```
func (c *AggregatorRpcClient) GetAggregatedCheckpointMessages(fromTimestamp, toTimestamp uint64) (*messages.CheckpointMessages, error) {
	c.logger.Info("Getting checkpoint messages from aggregator")

	type Args struct {
		FromTimestamp, ToTimestamp uint64
	}

	var checkpointMessages *messages.CheckpointMessages
	err := c.sendRequest(func() error {
		err := c.rpcClient.Call("Aggregator.GetAggregatedCheckpointMessages", &Args{fromTimestamp, toTimestamp}, checkpointMessages)
		if err != nil {
			c.logger.Info("Received error from aggregator", "err", err)
			return err
		}

		c.logger.Info("Checkpoint messages fetched from aggregator")
		return nil
	})

	return checkpointMessages, err
}
```

Reply:
Also noticed `databaser` fetch functions done in the same fashion. 
Such pattern usually applied when "filler" function agnostic to dst and just fills it, `void*` in C or` interface{}` in Go.

For example:
borsh: `func Deserialize(s interface{}, data []byte) error`
`void *memcpy(void *dest, const void * src, size_t n)`
In above the type needed specified outside the call but In cases of Fetch* functions or this one we already know a type. So there's no need for such semantic

Reply:
Done in [d32ac59](https://github.com/NethermindEth/near-sffl/pull/57/commits/d32ac59b490af6668132639963d4425c573560d8) and [36c7217](https://github.com/NethermindEth/near-sffl/pull/57/commits/36c721776cfde7b3012f6e463861e4ecba9c615c)